### PR TITLE
Feat/manager 15714 poweroff nutanix node

### PR DESCRIPTION
**Pull request by tibs245**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nutanix-scale-in-out`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-15714
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

File: packages/manager/modules/nutanix/src/dashboard/component/poweroff-modal/controller.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          )}`,
+        );
+      })
+      .catch((error) => {
+        this.handleError(
+          `${this.$translate.instant(
+            'nutanix_dashboard_poweroff_node_error_banner',
+          )} ${error.message}`,
+        );
+      });
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/component/poweroff-modal/controller.js:31]:
```suggestion
      }).finally( ()=> {
         this.isLoading = false;
      });
```

File: packages/manager/modules/nutanix/src/service.js [hunk 2]
```diff
@@ -305,6 +305,24 @@ export default class NutanixService {
     };
   }
 
+  getDedicatedNodePowerId(nodeId) {
+    return this.$http
+      .get(`/dedicated/server/${nodeId}/boot?bootType=power`)
+      .then(({ data }) => data[0]);
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/service.js:311]: What about if data is null or empty ?

File: packages/manager/modules/nutanix/src/service.js [hunk 3]
```diff
@@ -305,6 +305,24 @@ export default class NutanixService {
     };
   }
 
+  getDedicatedNodePowerId(nodeId) {
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/service.js:308]:
```suggestion
  getClusterNodePowerId(nodeId) {
```

File: packages/manager/modules/nutanix/src/service.js [hunk 4]
```diff
@@ -305,6 +305,24 @@ export default class NutanixService {
...
+    return this.$http
+      .get(`/dedicated/server/${nodeId}/boot?bootType=power`)
+      .then(({ data }) => data[0]);
+  }
+
+  updateDedicatedNodePowerStateOff(nodeId) {
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/service.js:314]:
```suggestion
  updateClusterNodePowerStateOff(nodeId) {
```

File: packages/manager/modules/nutanix/src/service.js (hunk 2 above)
Comment (tibs245) [reply packages/manager/modules/nutanix/src/service.js:311]: It's not possible. This mean the server has not poweroff status or server not exist

**Review by darsene — APPROVED** [review]

File: packages/manager/modules/nutanix/src/dashboard/component/poweroff-modal/template.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    data-secondary-action="$ctrl.goBack()"
+    data-secondary-label="{{:: 'nutanix_dashboard_poweroff_node_cancel' | translate }}"
+    data-heading="{{:: 'nutanix_dashboard_poweroff_node_title' | translate }}"
+>
+    <form name="$ctrl.confirmPoweroffForm" novalidate>
+        <oui-message data-type="warning" class="mb-2">
+            <div
+                data-translate="nutanix_dashboard_poweroff_node_description"
+            ></div>
+            <a class="oui-link_icon" href="{{$ctrl.poweroffGuidelink}}">
```
Comment (ThibaudCrespin) [inline packages/manager/modules/nutanix/src/dashboard/component/poweroff-modal/template.html:16]:
```suggestion
            <a class="oui-link_icon" data-ng-href="{{$ctrl.poweroffGuidelink}}">
```

**Review by ThibaudCrespin — APPROVED** [review]

**Review by vovh — APPROVED** [review]

**Pull request merged**